### PR TITLE
Change brokers references to bootstrap servers for Kafka event source

### PR DESCRIPTION
**Pull request by dubee**
Replaces references to `brokers` with `bootstrap servers` for consistency with Apache Kafka documentation.

### CODE REVIEW

File: contrib/kafka/pkg/apis/sources/v1alpha1/kafka_types.go [hunk 1]
```diff
@@ -66,9 +66,9 @@ type KafkaSourceNetSpec struct {
 
 // KafkaSourceSpec defines the desired state of the KafkaSource.
 type KafkaSourceSpec struct {
-	// Brokers are the Kafka servers the consumer will connect to.
+	// Bootstrap servers are the Kafka servers the consumer will connect to.
 	// +required
-	Brokers string `json:"brokers"`
+	BootstrapServers string `json:"bootstrapServers"`
```
Comment (evankanderson) [inline contrib/kafka/pkg/apis/sources/v1alpha1/kafka_types.go]: We haven't released this EventSource yet, correct?

Comment (dubee) [reply contrib/kafka/pkg/apis/sources/v1alpha1/kafka_types.go]: Nope, we haven't release it yet.

**Comment by evankanderson** [comment]
Whoops, looks like you have conflicts now.

**Comment by matzew** [comment]
@evankanderson looks like the conflicts are sorted

/lgtm

**Comment by matzew** [comment]
It's somewhat important this goes in, as suggested in the orignal PR. since the channel uses that too - and also the apache kafka upstream community uses that term.

**Pull request merged**